guides/common/modules/con_user-groups.adoc

```diff
@@ -3,6 +3,13 @@
 
 With {Project}, you can assign permissions to groups of users.
 You can also create user groups as collections of other user groups.
-If using an external authentication source, you can map {Project} user groups to external user groups as described in xref:Configuring_External_User_Groups_{context}[].
+
+ifdef::satellite[]
+If using an external authentication source, you can map {Project} user groups to external user groups as described in {InstallingServerDocURL}Configuring_External_User_Groups_satellite[Configuring external user groups] in _{InstallingServerDocTitle}_.
+endif::[]
+
+ifndef::satellite[]
+If using an external authentication source, you can map {Project} user groups to external user groups as described in {InstallingServerDocURL}Configuring_External_User_Groups_foreman[Configuring external user groups].
+endif::[]
```

Comment:
```suggestion
If using an external authentication source, you can map {Project} user groups to external user groups as described in {InstallingServerDocURL}Configuring_External_User_Groups_satellite[Configuring external user groups] in _{InstallingServerDocTitle}_.
```

Reply:
Thanks very much for the additional info. What confused me was the fact that the {project-context} translates into "admin" in the "Administering Red Hat Satellite" guide: see e.g. https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html-single/administering_red_hat_satellite/index#Starting_and_Stopping_Server_admin 

At any rate, I have implemented your suggestions (haven't accepted the ^^ commit as this would lead to a merge conflict):
- I have replaced the hard values with {project-context} in the links
- I have removed the ifdef/ifndef statements and only left the version that uses  ...DocURL and ...DocTitle
The title builds when I run "make BUILD=satellite" and "make BUILD=katello".

I hope the fork is ready to be merged to master now. Please let me know if not.